tests/cli/test_descriptor.py

```diff
@@ -0,0 +1,64 @@
+import os
+import numpy as np
+
+import click
+from click.testing import CliRunner
+
+import pytest
+from pytest import approx
+
+from ase import Atoms
+from ase.io import write, read
+
+from wfl.cli.cli import cli
+from wfl.configset import ConfigSet, OutputSpec
```

Comment:
Do we actually need ConfigSet/OutputSpec for this test?

Reply:
removed